src/NIST-800-53/rules/iam/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+export { default as nist80053IamGroupMembershipCheck } from './nist80053IAMGroupMembershipCheck';
+export { default as nist80053IamUserNoPoliciesCheck } from './nist80053IAMUserNoPoliciesCheck';
+export { default as nist80053IamNoInlinePolicyCheck } from './nist80053IAMNoInlinePolicyCheck';
```

Comment:
Matching naming style of other rules. File names should also be changed

```suggestion
export { default as nist80053IamGroupMembership } from './nist80053IAMGroupMembership';
export { default as nist80053IamUserNoPolicies } from './nist80053IAMUserNoPolicies';
export { default as nist80053IamNoInlinePolicy } from './nist80053IAMNoInlinePolicy';
```